builtin/providers/google/resource_sql_database_instance_test.go

```diff
@@ -408,66 +408,73 @@ func testAccCheckGoogleSqlDatabaseInstanceEquals(n string,
 			return fmt.Errorf("Error settings.pricing_plan mismatch, (%s, %s)", server, local)
 		}
 
-		if instance.ReplicaConfiguration != nil &&
-			instance.ReplicaConfiguration.MysqlReplicaConfiguration != nil {
-			server = instance.ReplicaConfiguration.MysqlReplicaConfiguration.CaCertificate
-			local = attributes["replica_configuration.0.ca_certificate"]
+		if instance.ReplicaConfiguration != nil {
+			server = strconv.FormatBool(instance.ReplicaConfiguration.FailoverTarget)
+			local = attributes["replica_configuration.0.failover_target"]
```

Comment:
attributes is a map[string]string, so we do `strconv.FormatBool` to get the returned value as a string and then do a string comparison.

There is no test that currently sets this property (the one that looks like it would set replica configuration doesn't actually run, it's just a variable that never gets called)